packages/ui/src/modules/onboarding/serp.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
wouldn't it make more sense to call this file `opera-serp`?

Reply:
Sure, I think we moved to `oper-serp` name after the initial code, so it is kind of leftover.